aptos-move/framework/aptos-framework/sources/multisig_account.move

```diff
@@ -1148,8 +1148,8 @@ module aptos_framework::multisig_account {
     use std::string::utf8;
     use std::features;
 
-    #[test_only]
-    const PAYLOAD: vector<u8> = vector[1, 2, 3];
+    // #[test_only]
+    // const PAYLOAD: vector<u8> = vector[1, 2, 3];
```

Comment:
As we discussed this should come with a TODO(#nnn) with linked bug.

Also you can replace with `payload()` for now for less impact.

Reply:
+1

Reply:
done